policy/modules/system/unconfined.te

```diff
@@ -63,6 +63,8 @@ ifdef(`init_systemd',`
 	init_pgm_spec_user_daemon_domain(unconfined_t)
 	allow unconfined_t self:system { status start stop reload };
 
+	systemd_use_passwd_agent_fds(unconfined_t)
```

Comment:
We have this: https://github.com/SELinuxProject/refpolicy/blob/1c0b2027f9d927f18734c49f58739121fd84eb5d/policy/modules/kernel/domain.te#L186

So this change seems unnecessary.